fifth-lesson/src/MetodTask18.java

```diff
@@ -0,0 +1,23 @@
+/*Найти в строке не только запятые, но и другие знаки препинания. Подсчитать
+общее их количество.*/
+
+
+public class MetodTask18 {
+    public static void main(String[] args) {
+        String str = " Строковый литерал — это последовательность символов, " +
+                "заключенных в двойные кавычки.&? "; // Исходная строка
+        int n = 0; //Счётчик символов
+        char[] symbolsCheck = {',', '.', '-', '?', '!', '|', '%', '/', '=', '_', '&'}; //Массив символов для поиска
```

Comment:
more efficient will be check char, number and space symbol - everything else will be what you are looking for

Reply:
I don't quite understand how in this example to check for char or number

Reply:
One way is to use util methods Character.isDigit and Character.isLetter. Another - regexp.